packages/core-js/internals/structured-clone.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable es/no-map -- safe */
+/* eslint-disable es/no-set -- safe */
+'use const';
```

Comment:
What is `'use const'`?

Reply:
lol, I intended to write `'use strict'`...